docs_versioned_docs/version-ros2humble/robots/outdoor_robots/jackal/maintenance_jackal.mdx

```diff
@@ -6,6 +6,7 @@ sidebar_position: 4
 
 import ComponentIntroductionJackal from "/components/introduction_jackal.mdx";
 import GettingNewPackages from "/components/maintenance/getting_new_packages.mdx";
+import JackalFirmware from "../../../components/_jackal_firmware.mdx";
```

Comment:
@hilary-luo are the existing absolute links acceptable? I recall you mentioning that we need to use relative links for Docusaurus versioning.

Reply:
For components it ends up being a mixture of relative and absolute. Any components in the global components folder (non-versioned) need to be absolute links, and components in the local versioned components folder need to be relative.